modules/ROOT/pages/migration/index.adoc

```diff
@@ -1,729 +1,1008 @@
+[source, graphql, indent=0]
+----
+type Tech @node(additionalLabels: ["TechDB"]) {
+  name: String
+}
+# becomes
+type Tech @node(labels: ["Tech", "TechDB"]) {
+  name: String
+}
+----
 
-This has all changed in `@neo4j/graphql`, with a `where` argument to select the node, and then an `update` argument (amongst many others) to select what to update.
+.Current equivalent to both arguments
+[source, graphql, indent=0]
+----
+type Tech @node(label: "TechDB", additionalLabels: ["AwesomeTech"]) {
+  name: String
+}
+# becomes
+type Tech @node(labels: ["TechDB", "AwesomeTech"]) {
+  name: String
+}
+----
 
-For example, updating the average rating of the Movie Forrest Gump:
+|`@queryOptions` and `limit`
+| Removed and moved to `@limit`.
+a|
+.Outdated example
+[source, graphql, indent=0]
+----
+type Record @queryOptions(limit: { default: 10, max: 100 }) {
+  id: ID!
+}
+----
 
+.Updated version using `@limit`
 [source, graphql, indent=0]
 ----
-mutation {
-    UpdateMovie(title: "Forrest Gump", averageRating: 5.0) {
-        title
-        averageRating
-    }
+type Record @limit(default: 10, max: 100) {
+  id: ID!
 }
 ----
 
-Will look like the following using the new library:
+|`@readonly` and `@writeonly` 
+|Removed and replaced by the `@selectable` and `@settable` directives.
+They can be used to configure not only if fields are readable or writable, but also when they should be readable or writable.
+a|* `@readonly` -> `@settable(onCreate: false, onUpdate: false)`
+* `@writeonly` -> `@selectable(onRead: false, onAggregate: false)`
 
+|`@query` and `@relationship`
+|Aggregation operations are no longer generated by default.
+They can be enabled case by case using the directives xref::/schema-configuration/type-configuration.adoc#_query[`@query`] and xref::/schema-configuration/field-configuration.adoc#_relationship[`@relationship`].
+a|
 [source, graphql, indent=0]
 ----
-mutation {
-    updateMovies(where: { title: "Forrest Gump" }, update: { averageRating: 5.0 }) {
-        movies {
-            title
-            averageRating
-        }
-    }
+type Movie {
+  title: String!
+}
+
+type Actor @query(aggregate: true) {
+  name: String!
+  actedIn: [Movie!]! @relationship(type: "ACTED_IN", direction: OUT, aggregate: true) 
 }
 ----
+|===
+
+[relationship-aggregate]
+== Relationship updates
 
-=== Deleting
+Here are the changes and updates to `@relationship`-related features.
 
-The arguments for selecting which node to delete have now been moved into a `where` argument.
+[discrete]
+=== *Relationship types are now automatically escaped*
 
-Additionally, the return value is now a `DeleteInfo` object informing how many nodes and relationships were deleted.
+Relationship types are now automatically escaped. 
+If you have previously escaped your relationship types using backticks, you must now remove these as this is covered by the library.
 
-For example, deleting a movie:
+[discrete]
+=== *`aggregate` value now defaults to false*
```

Comment:
Our bad, we changed the idea and we never released this, so defaults is still true!